### PR TITLE
Update _grade.py

### PR DESCRIPTION
**Pull request by rrroyal**
Check if `j.get("Kategoria")` exists before using it as value

### CODE REVIEW

File: vulcan/_grade.py [hunk 1]
```diff
@@ -80,7 +80,10 @@ def from_json(cls, j):
         id = j.get("Id")
         pracownik = Pracownik.from_json(j.get("Pracownik"))
         przedmiot = Przedmiot.from_json(j.get("Przedmiot"))
-        kategoria = KategoriaOceny.from_json(j.get("Kategoria"))
+        if (j.get("Kategoria")):
+            kategoria = KategoriaOceny.from_json(j.get("Kategoria"))
```
Comment (kapi2289) [inline vulcan/_grade.py]: You've checked that `Kategoria` exists so in this line you don't need to use `get`
```suggestion
            kategoria = KategoriaOceny.from_json(j["Kategoria"])
```

File: vulcan/_grade.py [hunk 2]
```diff
@@ -80,7 +80,10 @@ def from_json(cls, j):
...
+        else:
+            kategoria = ""
```
Comment (kapi2289) [inline vulcan/_grade.py]: I think it would be better to return `None` here
```suggestion
            kategoria = None
```

File: vulcan/_grade.py [hunk 3]
```diff
@@ -80,7 +80,10 @@ def from_json(cls, j):
         id = j.get("Id")
         pracownik = Pracownik.from_json(j.get("Pracownik"))
         przedmiot = Przedmiot.from_json(j.get("Przedmiot"))
-        kategoria = KategoriaOceny.from_json(j.get("Kategoria"))
+        if (j.get("Kategoria")):
```
Comment (kapi2289) [inline vulcan/_grade.py]: In Python, you cannot use parentheses in `if` statements.
```suggestion
        if j.get("Kategoria"):
```

**Comment by kapi2289** [comment]
Try to fix your mistakes and I'll approve the PR 😃

File: vulcan/_grade.py (hunk 3 above)
Comment (rrroyal) [reply vulcan/_grade.py]: I didn't actually knew that haha

**Comment by kapi2289** [comment]
Thanks for your help!

**Pull request merged**